libfwupd/fwupd.map

```diff
@@ -323,6 +323,8 @@ LIBFWUPD_1.2.5 {
 
 LIBFWUPD_1.2.6 {
   global:
+    fwupd_build_device_state_json;
```

Comment:
Does this need to be exported? It seems only used in `src/fu-tool.c` -- as soon as we export it it means we stuck to the JSON "API" forever.

Reply:
Not really, I moved it to the functions to `*-private.h`

Reply:
removed the wrong one?

Reply:
whoops, thanks will fix.

Reply:
Actually I'm actually a little confused where you want to see all this code live.  Are you meaning to move _all_ this stuff "out" of libfwupd and into `fu-tool`?

Reply:
well, fwupd_build_device_state_json could just be in fu-tool.c, no? I think it makes perfect sense to export the fwupd_device_to_json and fwupd_release_to_json, just not so much the other bits.

Reply:
Moved it over.